src/max_flow/maxFlow_process.cpp

```
@@ -0,0 +1,87 @@
+            which,
+            (*result_tuples), (*result_count),
+            log, notice, err);
+
+    auto name = std::string(" processing ") + pgrouting::get_name(which);
+    time_msg(name.c_str(), start_t, clock());
+
+    if (!err.str().empty() && (*result_tuples)) {
+        pfree(*result_tuples);
+        (*result_tuples) = nullptr;
+        (*result_count) = 0;
+    }
```

Cppcheck (2.19.0)

[style] The function 'pgr_free' is never used.

(unusedFunction)